crates/curp/src/server/raw_curp/mod.rs

```diff
@@ -1376,7 +1384,8 @@ impl<C: Command, RC: RoleChange> RawCurp<C, RC> {
                     .map_lock(|mut cst_l| _ = cst_l.config.remove(node_id));
                 self.lst.remove(node_id);
                 _ = self.ctx.sync_events.remove(&node_id);
-                let _ig = self.ctx.cluster_info.remove(&node_id);
+                let _ig1 = self.ctx.cluster_info.remove(&node_id);
```

Comment:
The log persistent and member info persistent is not atomic, could this cause inconsistent between log and the member info?

Reply:
I didn't think of a possible problem for a while, can you provide an example？

Reply:
I reconsidered about it, I think it's fine as long as the log is properly persistent and replayed because member info is persistent before log persistent. Resolved.